config/setup/browse.go

```diff
@@ -193,6 +193,10 @@ th a {
 		margin-top: 70px;
 	}
 }
+
+#name {
+	white-space: pre;
```

Comment:
@mholt testing `pre-wrap` and `pre` show's no difference with long names, however, W3C says:
```
pre:	Whitespace is preserved by the browser. Text will only wrap on line breaks. Acts like the <pre> tag in HTML
pre-wrap: 	Whitespace is preserved by the browser. Text will wrap when necessary, and on line breaks
```
I think we don't need text to wrap when it's "necessary"